_pandoc_filter/add_title.lua

```diff
@@ -0,0 +1,317 @@
+	content = string.gsub(content, '%(/verify%-receive%-address/coldcard%-advanced%)', '(#verify-receive-address-on-coldcard-advanced)')
+	content = string.gsub(content, '%(coldcard%-advanced%)', '(#verify-receive-address-on-coldcard-advanced)')
+	content = string.gsub(content, '%(/verify%-receive%-address/cobo%)', '(#verify-receive-address-on-cobo-vault)')
+	content = string.gsub(content, '%(/verify%-receive%-address/specter%)', '(#verify-receive-address-with-specter-desktop)')
+	content = string.gsub(content, '## Table of Contents', '')
+	content = string.gsub(content, '%(/known%-issues/seeds%-and%-privacy%)', '(#seeds-and-privacy)')
+	content = string.gsub(content, '%(/why%-multisig%-advanced#shamirs%-secret%-sharing%-scheme%)', '(#shamirs-secret-sharing-scheme)')
+	content = string.gsub(content, '%(/backup%-wallet/public%-keys%-advanced#extended%-public%-key%-info%)', '(#extended-public-key-info)')
+	content = string.gsub(content, '%(advanced%)', '(#emergency-recovery-advanced)')
+	content = string.gsub(content, '%(/known%-issues/hardware/trezor%)', '(#trezor)')
+	content = string.gsub(content, '%(/known%-issues/hardware/specter%-diy%)', '(#specter-diy2)')
+	content = string.gsub(content, '%(/known%-issues/hosted/unchained%)', '(#unchained-capital)')
+	content = string.gsub(content, '%(/known%-issues/hosted/casa%)', '(#casa)')
+	content = string.gsub(content, '%(/known%-issues/cost%)', '(#cost)')
+	content = string.gsub(content, '%(/known%-issues/complexity%)', '(#complexity)')
+	content = string.gsub(content, '%(/known%-issues/software/seedpicker%)', '(#seedpicker)')
+	content = string.gsub(content, '%(/backup%-wallet/seeds%)', '(#backup-seeds)')
+	content = string.gsub(content, '%(/setup%-wallets/%)', '(#setup-hardware-wallet-overview)')
+	content = string.gsub(content, '%(/setup%-wallets%)', '(#setup-hardware-wallet-overview)')
+	content = string.gsub(content, '%(/why%-multisig%)', '(#why-multisig)')
+	content = string.gsub(content, '%(/why%-multisig%-advanced%)', '(#why-multisig-advanced)')
+    content = string.gsub(content, '%(/assets/guide/bip39_wordlist.pdf%)', '(https://github.com/btcguide/btcguide.github.io/blob/master/assets/guide/bip39_wordlist.pdf)')
+    content = string.gsub(content, '%(/quorum%-advanced#3%-of%-5%-is%-excellent%)', '(#of-5-is-excellent)')
+    content = string.gsub(content, '%(advanced#redundant_address_verification%)', '(#verify-receive-address-advanced)')
+	content = string.gsub(content, '%(/backup%-wallet%)', '(#backup-wallet)')
+	content = string.gsub(content, '%(/backup%-wallet/seeds%)', '(#backup-seeds)')
+	content = string.gsub(content, '%(/backup%-wallet/public%-keys%)', '(#backup-public-keys)')
+
+	return content
+end
+
+
+
+
+function Para(elem)
+    if #elem.content == 1 and elem.content[1].t == "Image" then
+      local img = elem.content[1]    
+      if img.classes:find('markdown',1) then
```

Comment:
Can this be DRY?

Also, unclear what markdownskip and these #s are (might be clear if DRY).